# HG changeset patch
# User Rik <rik@octave.org>
# Date 1544725549 28800
#      Thu Dec 13 10:25:49 2018 -0800
# Node ID db43bd1fc038130a6e22ac619706f2ff58012c1f
# Parent  52a595c996bc39d58bb8024b7687b6b6cbf9a5d1
build: Add wiki files for release process.

* etc/RELEASE.BUG_FIX_LIST: New file with bug fix list template in wiki format.

* etc/RELEASE.CHECKLIST: Renamed from RELEASE.PROCESS.

diff --git a/etc/RELEASE.BUG_FIX_LIST b/etc/RELEASE.BUG_FIX_LIST
new file mode 100644
--- /dev/null
+++ b/etc/RELEASE.BUG_FIX_LIST
@@ -0,0 +1,24 @@
+The following bugs must be fixed prior to the next release.  See also the [[5.0.0 Release Checklist]].
+
+<p>Sign up for a bug by filling in the Owner field so people won't duplicate each other's work.</p>
+
+<p>Use <pre><strike> ... </strike></pre> to cross an item off the list when it has been done.</p>
+
+<p>Add <pre>WON'T FIX</pre> and the strike through tag for any bugs that will not be fixed in this round.</p>
+
+<div class="tocinline">__TOC__</div>
+
+----
+
+== Bugs marked as Crash ==
+
+== Bugs marked Configuration and Build System ==
+
+== Bugs with severity >= 4 ==
+
+== Bugs marked as regressions ==
+
+== Other Bugs ==
+
+== Potentially excluded bugs ==
+These bugs have been provisionally decided not to be blockers for 5.0.0.  That doesn't mean they can't be fixed, if someone wants to fix them.
diff --git a/etc/RELEASE.PROCESS b/etc/RELEASE.CHECKLIST
rename from etc/RELEASE.PROCESS
rename to etc/RELEASE.CHECKLIST
--- a/etc/RELEASE.PROCESS
+++ b/etc/RELEASE.CHECKLIST
@@ -1,18 +1,18 @@
 <!----------------------------------------------------------------------------->
 <! This file contains a list of steps to accomplish before producing a release.
    The format of the file is wiki markup and can be directly used as a page
-   on wiki.octave.org.
+   on wiki.octave.org. >
 <!----------------------------------------------------------------------------->
 ==5.0.0 Release Tasks==
 
 <!----------------------------------------------------------------------------->
 # Update gnulib to latest version
-# Must occur first as it could resolve existing, or create new, bug reports
+#: Must occur first as it could resolve existing, or create new, bug reports
 #: Completion Date:
 <!----------------------------------------------------------------------------->
 # File bug reports for all outstanding bugs known, but not reported
 #* Put out a general call for reports on Octave-Maintainers and Octave-Help list
 #: Completion Date:
 <!----------------------------------------------------------------------------->
 # Review patch tracker/bug list for any patches submitted that may be included before release
 #: Completion Date:
